Cargo.toml

```diff
@@ -12,18 +12,19 @@ readme = "README.md"
 
 
 [features]
+base64 = [ "base64-compat" ]
 fuzztarget = ["secp256k1/fuzztarget", "bitcoin_hashes/fuzztarget"]
 unstable = []
 rand = ["secp256k1/rand-std"]
 use-serde = ["serde", "bitcoin_hashes/serde", "secp256k1/serde"]
-secp-recovery = ["secp256k1/recovery"]
```

Comment:
This is a breaking change, you can add that to default features, but removing a feature is a breaking change

Reply:
Not sure if this has been released yet, @apoelstra ? It's only recently added by @RCasatta .

Reply:
I mean we can still release the `secp-recovery` feature in `0.25.1` and then remove it in `0.26.0`? Making it a default feature means that all the code in this PR needs to be feature-gated.

Reply:
@apoelstra says that it wasn't yet released, so you can restore this back? sorry.
I'm not sure if people still want this to be feature gated or not, personally I'm fine with it not feature gated.